src/js/components/Box.js

```diff
@@ -175,5 +177,6 @@ Box.defaultProps = {
   direction: 'column',
   pad: 'none',
   tag: 'div',
+  texturePosition: 'center top',
```

Comment:
When we were discussing this with @L0ZZI yesterday, the decision was that the default should be centered but that Grommet users should have the option to change the background position if they need to pin it elsewhere for the purpose of responsiveness.